app/assets/stylesheets/profiles.scss

```diff
@@ -0,0 +1,7 @@
+.user_avatar {
+  display: inline-block!important;
+}
+
+.file.optional.user_avatar {
```

Comment:
we're in scss, try to use nested styling syntax instead of regular CSS;
```scss
.file {
  .optional {
    .user_avatar {
      // styles goes here
    }
  }
}
```

Reply:
Roman it's different what you written to what Andriy did.
Fof me all is ok